### PR TITLE
fix(keyless-backup): small UI fix on status screen

### PR DESCRIPTION
**Pull request by satish-ravi**
### Description

[Figma](https://www.figma.com/file/gGp6SVFWMbnSltkGzNzrpC/Wallet-Recovery?node-id=3604%3A31336&mode=dev)

### Test plan

Manually

| Status | Before | After |
|--------|--------|--------|
| Completed |  |  |
| Failed |  |  | 

### Related issues

N/A

### Backwards compatibility

N/A

### CODE REVIEW

File: src/keylessBackup/KeylessBackupProgress.tsx [hunk 1]
```diff
@@ -1,22 +1,22 @@
+import { NativeStackScreenProps } from '@react-navigation/native-stack'
 import React, { useEffect } from 'react'
-import { StyleSheet, Text, BackHandler, View } from 'react-native'
 import { useTranslation } from 'react-i18next'
-import { NativeStackScreenProps } from '@react-navigation/native-stack'
-import { StackParamList } from 'src/navigator/types'
-import { Screens } from 'src/navigator/Screens'
+import { BackHandler, StyleSheet, Text, View } from 'react-native'
```
Comment (cajubelt) [inline src/keylessBackup/KeylessBackupProgress.tsx:4]: unrelated import diffs

**Review by cajubelt — APPROVED** [review]

File: src/keylessBackup/KeylessBackupProgress.tsx (hunk 1 above)
Comment (satish-ravi) [reply src/keylessBackup/KeylessBackupProgress.tsx:4]: yeah its the vscode sorter 😅
